README.md

```diff
@@ -1,34 +1,113 @@
 > By contributing to this project, you agree to abide by our [Code of Conduct](https://github.com/freedomofpress/.github/blob/main/CODE_OF_CONDUCT.md).
 
-# SecureDrop Debian Packaging
-
 [![CircleCI](https://circleci.com/gh/freedomofpress/securedrop-builder/tree/main.svg?style=svg)](https://circleci.com/gh/freedomofpress/securedrop-builder/tree/main)
 
```

Comment:
Is putting the badge above the main header intentional? (I notice it doesn't show up in the preview.)

Reply:
:/ I see it still when I "view file": https://github.com/freedomofpress/securedrop-builder/blob/1f5feba3f9176b17e16084bb7c0877dcd98560bc/README

The badge is intentionally there and this PR shouldn't remove it.